src/applications/find-va-forms/containers/SearchResults.jsx

```diff
@@ -1,7 +1,77 @@
+// Dependencies
 import React from 'react';
+import PropTypes from 'prop-types';
+import SortableTable from '@department-of-veterans-affairs/formation-react/SortableTable';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
Since this is just a bundle loaded up via code-splitting, I think it would be cool if we could get away without Lodash added into the bundle